contents/docs/product-analytics/autocapture.mdx

```diff
@@ -16,7 +16,7 @@ PostHog can capture frontend events automatically using autocapture. This captur
 
 This means you don't need to manually add tracking for individual components, links, buttons, divs, spans, or other parts of your product. Autocapture also provides data for [heatmaps](/docs/toolbar/heatmaps).
 
-Autocapture is available for our [JavaScript Web](/docs/libraries/js), [React](/docs/libraries/react), and [React Native](/docs/libraries/react-native) SDKs and is **enabled by default**.
+Autocapture is available by for our [JavaScript Web](/docs/libraries/js), [React](/docs/libraries/react), [React Native](/docs/libraries/react-native) and  [iOS](/docs/libraries/ios) SDKs and is **disabled by default**. 
```

Comment:
mmm I am a bit confused now, I know RN and iOS are disabled by default, but the docs mention it is enabled for JS and React.
Can you double-check this? maybe @pauldambra knows what's the default state.
I really recall reading this somewhere that is disabled for all SDKs because of PII risk etc

Reply:
default config in posthog-js includes:

```
    autocapture: true,
    rageclick: true,
    capture_pageview: true,
```

https://github.com/PostHog/posthog-js/blob/e86a694aa7834f4efc73e1a01a5221180b9e4adc/src/posthog-core.ts#L132-L134

Reply:
so, at least on web it's enabled by default :)

Reply:
and then we listen to the decide response which can disable autocapture with `autocapture_opt_out`